attest/core/src/ias/verify.rs

```diff
@@ -234,6 +233,19 @@ impl VerificationReportData {
 
         Ok(())
     }
+
+    /// Try and parse the timestamp string into a chrono object.
+    #[cfg(feature = "std")]
+    pub fn parse_timestamp(&self) -> Result<chrono::DateTime<chrono::Utc>, VerifyError> {
+        // Intel provides the timestamp as ISO8601 (compatible with RFC3339) but without the
```

Comment:
If Intel is close to RFC3339 but not exactly to spec because of the "Z", would it make more sense to parse from string with a fmt specifier? 

https://docs.rs/chrono/0.4.11/chrono/struct.DateTime.html#method.parse_from_str

The chrono library does seem to support emitting a "Z" as an option for rfc3339...

https://docs.rs/chrono/0.4.11/chrono/struct.DateTime.html#method.to_rfc3339_opts

Reply:
Emitting it doesn't work, it needs a `+<offset>` in that case. I tried. I am not sure whats wrong with adding `Z` but we can also using `parse_from_str` if anyone feels strongly about it.